modules/optimization/test/tests/reporters/multiExperiment/tests

```diff
@@ -1,17 +1,15 @@
 [Tests]
   issues = '#21885 #26530'
-  design = 'VectorOfVectorRowSum.md VectorDotProduct.md VectorSum.md'
+  design = 'ParsedVectorReporter.md ParsedScalarReporter.md MultiAppReporterTransfer.md'
   [sampler_opt]
     requirement = "The system shall be able to find an optimal value to a linear combination of parallel forward problems run using the sampler system."
     type = CSVDiff
-    input = main.i
-    csvdiff = main_out.csv
+    input = optimize.i
+    csvdiff = optimize_out.csv
     min_parallel = 6
     max_parallel = 6
     max_threads = 1
     # skip recover for steady solve
     recover = false
-    # skip test if test is being run out-of-tree. Issue Ref: #25279
-    installation_type = in_tree
```

Comment:
did you fix that? which file did you modify?

Reply:
I deleted it.  I don't what it does but I don't think it was ever needed.  I copied that line from another test file that tested similar objects to what I was originally using in this problem before I changed it over to these new methods.

Reply:
This is used for tests that code up paths and don't run unless when run from the expected location 

To check if it s needed, run the test from outside the usual location. 
For example from an app do 

./run_tests --spec-file ../moose/modules/optimization

Reply:
I'm not sure what you mean but I did this and all tests pass:
From this directory:
/Users/mundlb/projects/moose/modules/optimization/test/tests/reporters/multiExperiment
I ran this command: > ../../../../run_tests -j 8 --spec-file=../../../../

Normally I would execute run_tests from here /Users/mundlb/projects/moose/modules/optimization

If I run just ../../../../run_test, it will only run the test in the directory I am in.  So I think I am testing that it does not need installation_type = in_tree

Reply:
I removed all of the installation_type = in_tree
and then I ran these tests from the combined app  /Users/mundlb/projects/moose/modules/combined
with
./run_tests -j 8 --spec-file /Users/mundlb/projects/moose/modules/optimization
And all of the optimization tests pass.  So I think this means in_tree was not needed.

Reply:
should be fine. If not we ll learn something